Meditation-App/style.css

```diff
@@ -0,0 +1,194 @@
+* {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+  }
+  .wrapper {
+    width: 100%;
+    height: 100vh;
+    overflow: hidden;
+  }
+  .buttons-minutes {
+    width: 100px;
+    height: 400px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    position: absolute;
+    top: calc(50% - 200px);
+    left: 10%;
+  }
+  
+  .buttons-minutes__button {
+    width: 100%;
+    height: 10%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition: all 0.3s;
+    background-color: transparent;
+    color: rgb(131, 131, 131);
+    font: 16px Arial;
+  }
+  .buttons-minutes__button:hover {
+    transition: all 0.3s;
+    background: rgba(255, 255, 255, 0.24);
+  }
+  .timer {
+    width: 200px;
+    height: 200px;
+    position: absolute;
+    border-radius: 50%;
+    top: calc(50% - 100px);
+    left: calc(50% - 100px);
+    background-color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    overflow: hidden;
+  }
+  .mask-left {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 50%;
+    height: 100%;
+    background-color: white;
+    z-index: 2;
+  }
+  .mask-right {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 50%;
+    height: 100%;
+    background-color: rgb(60, 73, 146);
+    visibility: hidden;
+  }
+  .timer__display {
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 40px;
+    background-color: rgb(145, 161, 255);
+    z-index: 5;
+  }
+  .timer__line {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+  }
+  .mask-line {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 50%;
+    height: 100%;
+    background-color: rgb(60, 73, 146);
+  }
+  @keyframes line {
+    0% {
+    }
+    100% {
+      transform: rotate(360deg);
+    }
+  }
+  @keyframes mask-left {
+    0% {
+      visibility: visible;
+    }
+    50%,
+    100% {
+      visibility: hidden;
+    }
+  }
+  @keyframes mask-right {
+    0% {
+      visibility: hidden;
+    }
+    50%,
+    100% {
+      visibility: visible;
+    }
+  }
+  
+  .buttons-weather {
+    width: 55px;
+    height: 300px;
+    position: absolute;
+    top: calc(50% - 150px);
+    right: 10%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+  }
+  .button {
+    background: transparent;
+    width: 55px;
+    height: 55px;
+    border-radius: 50%;
+    background-repeat: round;
+  }
+  img {
```

Comment:
common styles will be better keep at the start of the file

Reply:
Thanks, fix